src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+
+    public abstract double getArea();
+
```

Comment:
lines
All Methods in interface are abstract by default
```suggestion
    public double getArea();
```